### PR TITLE
Add custom protocol

### PR DESCRIPTION
**Pull request by Igor-lkm**
We are able to set custom host `webhookValidator({ host })`. 

It make sense also to have an option to set a protocol. E.g. when endpoint is behind a proxy (TLS termination proxy).

Example:
```
webhookValidator({
  host: ctx.header['x-forwarded-host'],
  protocol: ctx.header['x-forwarded-proto']
});
``` 

This PR would add an optional protocol.

### CODE REVIEW

File: test/webhookValidator.test.ts [hunk 1]
```diff
@@ -52,8 +52,11 @@ async function createTestService(
   );
   app.use(router.routes());
 
+  const protocol = options ? options.protocol || 'http' : 'http';
```
Comment (0xch4z) [inline test/webhookValidator.test.ts]: I'd prefer

```typescript
const protocol = options && options.protocol || 'http';
```

File: README.md [hunk 2]
```diff
@@ -33,3 +33,13 @@ router.post(
 
 app.use(router.routes()).listen(3000);
 ```
+
+# Example usage with custom host and port (e.g. behind proxy)
```
Comment (0xch4z) [inline README.md:37]: Thanks for updating the docs 😄

**Review by 0xch4z — CHANGES_REQUESTED** [review]
Thanks! This looks great, I just had one piece of stylistic feedback.

File: test/webhookValidator.test.ts (hunk 1 above)
Comment (Igor-lkm) [reply test/webhookValidator.test.ts]: Done!

**Review by 0xch4z — APPROVED** [review]

**Comment by 0xch4z** [comment]
@Igor-lkm This has been released as part of 1.2.0.

**Pull request merged**